ui/src/alerting/actions/notifications/endpoints.ts

```diff
@@ -59,75 +69,92 @@ export const getEndpoints = () => async (
     })
   } catch (e) {
     console.error(e)
+    dispatch(notify(copy.getEndpointsFailed(e.message)))
     dispatch({type: 'SET_ALL_ENDPOINTS', status: RemoteDataState.Error})
   }
 }
 
 export const createEndpoint = (data: NotificationEndpoint) => async (
-  dispatch: Dispatch<Action>
+  dispatch: Dispatch<Action | NotificationAction>
 ) => {
-  const resp = await api.postNotificationEndpoint({data})
+  try {
```

Comment:
This `try / catch` block was left out before so that the endpoint creation form could catch any errors and display an error message inside the form, rather than as a notification. I think that's a nicer UX and would like to keep it—is there any reason this needs to catch errors and display a notification?

It looks like `cloneEndpoint` would still have the same catch behavior if you removed the `try/catch` wrapping, since it also catches and displays a notification.

Related: https://github.com/influxdata/influxdb/issues/14609

Reply:
Thanks so much- I thought it was an oversight. will remove.

Reply:
https://github.com/influxdata/influxdb/issues/14790

Reply:
removing from add and edit keeping notifications for other things